### ISSUE TITLE
pix2pix: What is the output fake_A in the combined model for?

### ISSUE THREAD

**Issue by raibosome** [eriklindernoren/Keras-GAN#90]
I notice that `fake_A` is passed to the combined model as one of the outputs [here](https://github.com/eriklindernoren/Keras-GAN/blob/fcddb2186f9343a29b5f9d51f4d98ef392f0905f/pix2pix/pix2pix.py#L70). Is it necessary to have it there, or am I missing something?

**Comment by raibosome** [comment]
Pardon me, I figured it out. In the combined model, we are optimising towards 2 outputs - `valid` and `fake_A`. And their respective loss functions to minimise are `mse` and `mae`.
